### ISSUE TITLE
MelonDS Vulkan Support?

### ISSUE THREAD

**Issue by chrisleigh-stack** [rafaelvcaetano/melonDS-android#1280]
* **Device model:** <insert your device's model here>
* **Android version:** <insert your device's Android version here>

**Problem:**  
Write a small description of the problem.

**Steps to reproduce (if applicable):**
1. Write a list of steps
2. Required for the problem
3. To appear

Add screenshots if the issue can be visualized.

**Comment by LutraVibes** [comment]
I think this would be a great feature, but why would it be useful?

**Comment by WinUser-Generic** [comment]
More compatible devices at decent speed

**Comment by Fatcat2024-cpu** [comment]
This won't happen anytime soon

see ~ https://www.reddit.com/r/emulation/comments/9rytsz/the_ds_gpu_and_its_fun_quirks/
https://www.reddit.com/r/emulation/comments/n2hxi7/melonds_introducing_the_compute_renderer/

**Comment by Fatcat2024-cpu** [comment]
This won't happen anytime soon

see ~ https://www.reddit.com/r/emulation/comments/9rytsz/the_ds_gpu_and_its_fun_quirks/
https://www.reddit.com/r/emulation/comments/n2hxi7/melonds_introducing_the_compute_renderer/

**Comment by Unknowncat1991** [comment]
> No news ?

https://www.patreon.com/posts/melonds-android-146892224

**Comment by SapphireRhodonite** [comment]
Yup, this will take a lot of work; this is something that will take months and months of work.

> No news ?
